sdk/js/src/nft_bridge/__tests__/integration.ts

```diff
@@ -59,6 +59,7 @@ import {
 import { postVaaSolanaWithRetry } from "../../solana";
 import { tryNativeToUint8Array } from "../../utils";
 import { arrayify } from "ethers/lib/utils";
+import { parseVaa } from "../../vaa/wormhole";
 const ERC721 = require("@openzeppelin/contracts/build/contracts/ERC721PresetMinterPauserAutoId.json");
 
 setDefaultWasm("node");
```

Comment:
can this be removed too?

Reply:
`setDefaultWasm` will eventually be removed once the Token Bridge and NFT Bridge wasm imports are completely swapped out with the wasmless calls. Soon!

Reply:
What a day it will be!